python/lsst/pipe/tasks/snapCombine.py

```diff
@@ -352,5 +356,5 @@ def makeInitialPsf(self, exposure, fwhmPix=None):
         model = self.config.initialPsf.model
         self.log.info("installInitialPsf fwhm=%s pixels; size=%s pixels", fwhmPix, size)
         psfCls = getattr(measAlg, model + "Psf")
-        psf = psfCls(size, size, fwhmPix/(2.0*num.sqrt(2*num.log(2.0))))
+        psf = psfCls(size, size, fwhmPix / (2.0 * num.sqrt(2 * num.log(2.0))))
```

Comment:
We don't yet run `black` on pipe_tasks so tend to shy away from formatting commits such as this one which conform to the non-black style guide.